src/pages/main_layout.cr

```diff
@@ -21,35 +21,109 @@ abstract class MainLayout
+                link "Get Started", Guides::GettingStarted::Installing, class: "btn w-full sm:w-auto mt-6 sm:mt-0"
+              end
             end
           end
         end
-        render_main_content
+        render_feature_grid
+        render_freeform_text
       end
     end
   end
 
-  private def render_main_content
-    div class: "container mx-auto px-6" do
+  private def render_feature_grid
+    div class: "container text-center md:text-left mx-auto px-6 py-5 bg-white md:bg-transparent border-b border-grey-light md:border-none" do
       mount Shared::FlashMessages.new(@context.flash)
-      # TODO: Add real page content later
-      # content
+      div class: "flex flex-col md:flex-row md:bg-white mx-auto md:rounded-lg md:shadow md:-mt-12 md:px-5 md:py-12" do
+        content_block "Say goodbye to slow", <<-TEXT
+        A simple hello world response is returned in 0.1ms. Rendering
+        complex JSON usually takes only a few milliseconds.
+        TEXT
+
+        content_block "Batteries included", <<-TEXT
+        Authentication, asset management, CORS, database ORM, and more can
+        all be included when creating a new Lucky project.
+        TEXT
+
+        content_block "Fewer bugs. More joy", <<-TEXT
+        Instead of finding bugs in QA or in production, Lucky is designed to
+        catch as many bugs as possible at compile time.
+        TEXT
+      end
+    end
+  end
+
+  private def content_block(title, body)
+    div class: "w-full md:w-1/3 sm:px-0 md:px-8 my-5 md:my-0" do
+      h3 title, class: "mb-3 text-black font-bold text-base"
+      para body, class: "leading-loose text-sm text-grey-darker"
+    end
+  end
+
+  private def render_freeform_text
+    div class: "container mx-auto px-6 lg:px-32 markdown-content my-16" do
+      div class: "lg:px-16" do
+        raw CustomMarkdownRenderer.render_to_html <<-TEXT
+        ### Why is modern web development so complicated?
+
+        **There are more choices in tech than ever**, and with it comes more questions and more complexity.
+
+        Have you ever thought:
+
+        * "Why do I have to write **so much boilerplate**?"
+        * "How do **so many bugs make it to production**?"
+        * "Why is my app **so slow**!?"
+        * "Why do I have to choose between fast prototyping and long term maintainability?"
+
+        Every performance issue and every bug steals precious time from making your service better.
+
+        ### What if you could get something up quickly *and* maintain it for years to come?
+
+        Imagine this:
+
+        * Generators for quick code creation
+        * Built in authentication (Optional)
+        * Secure by default
+        * Built in Webpack config to get a SPA off the ground fast (Optional)
+        * Insanely fast execution so users aren’t left waiting
+        * Powerful type system that helps catch bugs before they ever hit production
```

Comment:
This is all true, but it feels a bit more "tell" not "show". Have you considered screenshots/short gifs of code that show things like trying to compile with bad types, running `lucky gen.action`, fast execution times, etc? I think those would be more impactful, even if you had a couple up top and then had a longer bulleted list below.